content/pl/build-release-run.md

```diff
@@ -0,0 +1,18 @@
+## V. Buduj, wypuszczaj, uruchamiaj
+### Precyzyjnie oddziel etap budowania od uruchamiania
+
+[Codebase](./codebase) jest przetwarzany w (nie developerskie) wdrożenie w trzech etapach
+
+* Podczas *etapu budowania* kod z repozytorium jest konwertowany do wykonywalnej paczki tzw. *buildu*. Używając wersji kodu zdefiniowanej przez commit w procesie deploymentu, w etapie budowania aplikacji pobiera się, dołącza do projektu [zależności](./dependencies) i kompiluje niezbędne zasoby.
+* W *etapie wypuszczania* aplikacji brany jest build stworzony w poprzednim etapie i konfigurowany na bazie [ustawień](./config) obecnego wdrożenia.  Stworzony w ten sposób *release* zawiera zbudowane źródło kodu, jego konfigurację i jest gotowe do uruchomienia w wybranym środowisku.
+* *Etap uruchamiania* (znany również jako "runtime") odpala aplikację w środowisku wykonawczym przez uruchomienie zestawu [procesów](./processes) opisanych w przygotowanym releasie.
+
+![Kod staje się buildem, jeśli zostanie połączony z konfiguracją by stworzyć release](/images/release.png)
+
+**Aplikacja dwunastu aspektów ścieśle rozgranicza etapy budowy, wydania i uruchomienia**  Nie jest więc możliwe zrobić zmian kodu podczas działania aplikacji, ponieważ nie można przenieść ich z powrotem do etapu budowy.
+
+Narzędzia do obsługi wdrożeń zazwyczaj oferują moduły do zarządzania releasami, w tym możliwość do powrotu do poprzedniej wersji (roll back). Np. narzędzie [Capistrano](https://github.com/capistrano/capistrano/wiki) przechowuje releasy w podkatalogu `releases`, gdzie obecna wersja releasu jest symlinkowana do tej z katalogu. Komenda `rollback` pozwala na szybką zmianę  wypuszczonej aplikacji na jedną z poprzednich.
+
+Każdy release powinien zawsze posiadać unikalny identyfikator, jak np. timestamp wypuszczenia aplikacji (taki jak `2011-04-06-20:32:17`) lub inkrementowany numer (np. `v100`). Do rejestru wypuszczonych wersji aplikacji można jedynie dodawać jej nowe wersje, a jego zawartość nie może być zmieniana w żaden inny sposób. Jedynym sposobem na zmianę wypuszczonej aplikacji wypuszczenie nowej.
+
+Tworzenie buildu aplikacji jest inicjowane przez developera, zawsze kiedy wdrażany jest nowy kod. Uruchomienie aplikacji może natomiast nastąpić automatycznie po restarcie serwera lub jednego z procesów aplikacji po błędzie krytycznym. Dlatego też etap uruchamiania powinien być jak najbardziej jednolity, nie stwarzając wielu możliwych problemów ze startem aplikacji, które mogą spowodować, że przestanie ona działać w środku nocy, gdy pod ręką nie będzie żadnego developera. Etap budowy aplikacji może być bardziej złożony, ponieważ ewentualne błędy są zawsze widoczne dla developera, który nadzoruje deploy aplikacji.
```

Comment:
1. Brak przecinka pomiędzy `developera` a `zawsze`
2. `s/  
Dlatego też etap uruchamiania powinien być jak najbardziej jednolity, nie stwarzając wielu możliwych problemów ze startem aplikacji, które mogą spowodować, że przestanie ona działać w środku nocy, gdy pod ręką nie będzie żadnego developera/  
Dlatego też etap uruchamiania aplikacji powinien być jak najbardziej jednolity minimalizując równocześnie ryzyko wystąpienia problemów ze startem aplikacji - mogą one spowodować zaprzestanie działania aplikacji np. w nocy, kiedy to nie ma nikogo "pod ręką".`